### DISCUSSION TITLE
Terminal window in SOC Administration section

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#2474] [Ideas] [answered]
It would be nice if there were access to a Terminal built right into the SOC webpage since there's so much manual configurations needed for SO. Multiple authentication requests are already built-in before a person could enter any commands, so I don't think it's necessarily a security issue.  Seems easy enough to add using pre-made things that are available.  Could add a True/False option into sls files to make it available to those who want it turned on.

**Comment by weslambert** [comment, marked as answer]
Thanks for the feedback! The overall goal here is to abstract most administrative tasks into a web interface, so you should rarely need to drop to a CLI, and RBAC will help further to limit certain potentially destructive tasks vs every day management tasks from the web interface.
